src/screens/Notifications/NotificationScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import { CompositeNavigationProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, Alert, TouchableOpacity, FlatList } from 'react-native';
+
+import { images } from 'app/assets';
+import { Header, ScreenTemplate, Button, FlatButton, ButtonType, Image } from 'app/components';
+import { Route, MainCardStackNavigatorParams, RootStackParams, Wallet } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: CompositeNavigationProp<
+    StackNavigationProp<RootStackParams, Route.MainCardStackNavigator>,
+    StackNavigationProp<MainCardStackNavigatorParams, Route.Notifications>
+  >;
+  wallets: any;
+}
+export class NotificationScreen extends Component<Props> {
+  onChangeEmailPress = () => {
+    Alert.alert('dsadasd');
+  };
+  onDeletePress = () => {};
+
+  onAddEmailPress = () => {
+    this.props.navigation.navigate(Route.AddEmail);
+  };
+
+  renderItem = (item: any) => {
```

Comment:
Must it be `any`? If we can't have a type, it'd be better to change it to `unknown` I think